src/main/java/com/hubspot/jinjava/lib/filter/DivideFilter.java

```diff
@@ -41,11 +42,11 @@
 public class DivideFilter implements Filter {
 
   @Override
-  public Object filter(Object object, JinjavaInterpreter interpreter, String... arg) {
+  public Object filter(Object object, JinjavaInterpreter interpreter, Object... arg) {
     if (arg.length < 1) {
       throw new TemplateSyntaxException(interpreter, getName(), "requires 1 number (divisor) argument");
     }
-    String toMul = arg[0];
+    String toMul = arg[0] == null ? null : Objects.toString(arg[0]);
```

Comment:
`Objects.toString` takes care of nulls for you.

Reply:
[Docs](https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html) state that it returns `"null"` and not `null`.
Also if I use `Objects.toString(arg[0])` then the [following test](https://github.com/HubSpot/jinjava/blob/master/src/test/java/com/hubspot/jinjava/lib/filter/AdvancedFilterTest.java#L57) breaks. Therefore had to ensure that `arg[0] == null` check was made.

Reply:
Ah, good to know.